### PR TITLE
Updated models

### PR DESCRIPTION
**Pull request by OlenaHatala**
Add function updateBoard, fix creating tickets, update Board and User models

### CODE REVIEW

File: back-end/controllers/boardController.js [hunk 1]
```diff
@@ -147,33 +154,181 @@ const create  = asyncHandler(async (req, res) => {
...
     } catch (error) {
         res.status(400).json({
             message: "Board was not created",
             error: error.message,
         })
+    }
+})
+
+
+
```
Comment (daftkid) [inline back-end/controllers/boardController.js:166]: Too much empty lines here

File: back-end/controllers/boardController.js [hunk 2]
```diff
@@ -17,11 +12,31 @@ const week_index = {
...
+        !(Arrays.equals(a.days[day].hours, b.days[day].hours))) {
+            return false   
+        }
+    }
 
-    const tickets = [{
+    return true
+}
+
+
```
Comment (daftkid) [inline back-end/controllers/boardController.js:32]: I suppose, this line is an extra one, should it be removed then?

File: back-end/controllers/boardController.js [hunk 3]
```diff
@@ -147,33 +154,181 @@ const create  = asyncHandler(async (req, res) => {
...
 
+
 const deleteBoard = async (req, res) => {
     const { id } = req.body;
 
     if (!id) {
         return res.status(400).json({ message: 'Board ID required' })
     }
-
+
```
Comment (daftkid) [inline back-end/controllers/boardController.js:303]: Tab space here is not needed

File: back-end/models/User.js [hunk 4]
```diff
@@ -22,6 +22,20 @@ const UserSchema = new Mongoose.Schema({
       minlength: 8,
       required: true,
     },
+    /////////
```
Comment (daftkid) [inline back-end/models/User.js:25]: wtf? :D

**Review by daftkid — CHANGES_REQUESTED** [review]
Also, please add .DS_Store file to the gitignore file in the root of the project, this file just contains some metadata generated by MacOS and does not have anything to do with your app

**Review by MariaPadalka — APPROVED** [review]

**Review by daftkid — APPROVED** [review]

**Review by MariaPadalka — APPROVED** [review]

**Pull request merged**